### PR TITLE
Fixed failures is few commands - v4

### PR DESCRIPTION
**Pull request by Pavithra1602**
Fixed failues in --build, --batch and --tmp commands.

Signed-off-by: Pavithra <pavrampu@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by Pavithra1602** [comment]
[root@ltc-zz8-lp4 ras]# avocado run sosreport.py
JOB ID     : 5785e424e06681373e363cc9315eddf4808455b4
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-10-20T05.02-5785e42/job.log
 (1/6) sosreport.py:Sosreport.test_short: PASS (954.48 s)
 (2/6) sosreport.py:Sosreport.test_user: PASS (239.67 s)
 (3/6) sosreport.py:Sosreport.test_plugins: PASS (721.10 s)
 (4/6) sosreport.py:Sosreport.test_others: PASS (1156.32 s)
 (5/6) sosreport.py:Sosreport.test_archive: PASS (986.15 s)
 (6/6) sosreport.py:Sosreport.test_PPC: PASS (127.94 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2020-10-20T05.02-5785e42/results.html
JOB TIME   : 4187.13 s

File: ras/sosreport.py [hunk 1]
```diff
@@ -38,6 +38,12 @@ def run_cmd_out(cmd):
         return process.system_output(cmd, shell=True, ignore_status=True,
                                      sudo=True).decode("utf-8")
 
+    def run_cmd_search(self, cmd, search_str):
+        self.output = self.run_cmd_out(cmd)
+        for line in reversed(self.output.splitlines()):
+            if search_str in line:
+                return line.strip()
```
Comment (harish-24) [inline ras/sosreport.py:45]: any specific reason `line[0]` is changed?

Comment (Pavithra1602) [reply ras/sosreport.py:45]: yes line[0] had empty value. .strip is removing it.

**Review by harish-24 — APPROVED** [review]
LGTM

**Pull request merged**